var/spack/repos/builtin/packages/damaris/package.py

```diff
@@ -13,80 +13,72 @@ class Damaris(CMakePackage):
 
     homepage = "https://project.inria.fr/damaris/"
     git = "https://gitlab.inria.fr/Damaris/damaris.git"
-    maintainers("jcbowden")
-
-    version("master", branch="master")
-    version("1.6.0", tag="v1.6.0")
-    version("1.5.0", tag="v1.5.0")
-    version("1.3.3", tag="v1.3.3")
-    version("1.3.2", tag="v1.3.2")
-    version("1.3.1", tag="v1.3.1")
-
-    variant("fortran", default=True, description="Enables Fortran support")
-    variant("hdf5", default=False, description="Enables the HDF5 storage plugin")
-    variant("static", default=False, description="Builds a static version of the library")
-    variant("catalyst", default=False, description="Enables the Catalyst visualization plugin")
-    variant("visit", default=False, description="Enables the VisIt visualization plugin")
-    variant(
-        "examples",
-        default=False,
-        description="Enables compilation and installation of the examples code",
-    )
-    variant("docs", default=False, description="Enables the building of dOxygen documentation")
-    variant(
-        "python",
-        default=False,
-        description="Enables building of Python enabled Damaris library - "
-        "boost::python boost::numpy needed",
-    )
-
-    depends_on("mpi")
-    depends_on("cmake@3.18.0:", type=("build"))
-    depends_on(
-        "boost"
-        "+exception+locale+system+serialization+chrono+atomic"
-        "+container+regex+thread+log+filesystem+date_time"
-        "@1.67:"
-    )
-    depends_on("xsd")
-    depends_on("xerces-c")
-    depends_on("hdf5@1.8.20:", when="+hdf5")
-    depends_on("paraview+python", when="+catalyst")
-    depends_on("visit+mpi", when="+visit")
-    depends_on("boost+thread+log+filesystem+date_time+python+numpy @1.67:", when="+python")
+    maintainers = ['jcbowden']
```

Comment:
Please change this back to the new maintainers syntax/directive (see https://spack.readthedocs.io/en/latest/packaging_guide.html#maintainers).